sysdeps/managarm/aarch64/signals.S

```diff
@@ -0,0 +1,8 @@
+
+.section .text
+.global __mlibc_signal_restore
+__mlibc_signal_restore:
+	ldr x0, =0x80000006
+	svc 0
+	brk #1
```

Comment:
Is this tested?

Reply:
This can't really be tested until we get posix-subsystem running, but it's just equivalent to the x86_64 code (supercall 6 and `__builtin_trap()`)